app/views/shared/_activity_menu.html.haml

```diff
@@ -34,6 +34,13 @@
           %i.fa.fa-times
           %span.text
             Close
+      - if current_user && @activity.layout == LightweightActivity::LAYOUT_SINGLE_PAGE && (current_user.is_admin || @activity.user.email == current_user.email)
+        %li.am-print
+          %a{:href => preview_activity_path(@activity,:print => true), :target => "_blank", :title => "Print a blank copy"}
```

Comment:
After the meeting today we decided that print param should continue to be supported. This way external sites can point directly at a URL which will load the page and immediately bring up the print dialog.

I now notice that you are using `preview_activity_path`, does that work ok for students?